admin-ui/plugins/services/Components/Configuration/LdapEditPage.js

```diff
@@ -5,14 +5,15 @@ import { Container, CardBody, Card } from 'Components'
 import LdapForm from './LdapForm'
 import { editLdap } from 'Plugins/services/redux/actions/LdapActions'
 import { buildPayload } from 'Utils/PermChecker'
+import { isEmpty } from 'lodash'
```

Comment:
Could you confirm if we are following this `lodash` instaution `https://github.com/GluuFederation/flex/issues/192`

Reply:
yes @duttarnab . [Imported using named aliases](https://github.com/GluuFederation/flex/pull/199/files#diff-60b1974d1e1714de933017c4b586cb861ea7775e04600880ca14d5bafddec57a)